roles/translator/src/lib/downstream_sv1/diff_management.rs

```diff
@@ -320,7 +320,7 @@ mod test {
     #[test]
     fn test_diff_management() {
         let expected_shares_per_minute = 1000.0;
-        let total_run_time = std::time::Duration::from_secs(11);
+        let total_run_time = std::time::Duration::from_secs(30);
         let initial_nominal_hashrate = measure_hashrate(5);
```

Comment:
@GitGab19 mentioned on our call that we could try to increase `5` to improve things